lawnchair/src/app/lawnchair/search/LawnchairAppSearchAlgorithm.kt

```diff
@@ -149,6 +162,9 @@ class LawnchairAppSearchAlgorithm(context: Context) : LawnchairSearchAlgorithm(c
         return createSearchTarget(id, action, extras)
     }
 
+    private fun Sequence<AppInfo>.showHiddenApps(): Sequence<AppInfo> =
```

Comment:
I think `filterHiddenApps()` describes this better

Reply:
13d518c6f2d0f296915bc3de53afa4dfcb900419